CHANGELOG.md

```diff
@@ -8,12 +8,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
         
 ### Changed
+
 - `tt build` now hides building output if `-V` is not provided.
 
 ### Fixed
 
 - ``tt start`` now does not start an instance if it is already running.
 
+### Added
+
+- smart auto-completion for `tt start`, `tt stop`, `tt restart`, `tt connect`, `tt build`, `tt clean`, `tt logrotate`, `tt status`.   
```

Comment:
Please, add notes to readme (autocompletion section) about smart auto-completion for app names and instances.

Reply:
Added.